src/SDK/Language/Kotlin.php

```diff
@@ -296,6 +279,212 @@ public function getParamExample(array $param): string
         return $output;
     }
 
+    /**
+     * Generate Kotlin-style map initialization
+     *
+     * @param array $data
+     * @return string
+     */
+    protected function getKotlinMapExample(array $data): string
+    {
+        return $this->getKotlinMapExampleRecursive($data, 0);
+    }
+
+    /**
+     * Recursive helper for generating Kotlin mapOf() with proper indentation
+     *
+     * @param array $data
+     * @param int $indentLevel Indentation level for nested maps
+     * @return string
+     */
+    private function getKotlinMapExampleRecursive(array $data, int $indentLevel): string
+    {
+        $mapEntries = [];
+        $baseIndent = str_repeat('    ', $indentLevel + 2);
+
+        foreach ($data as $key => $value) {
+            $formattedKey = '"' . $key . '"';
+            if (is_string($value)) {
+                $formattedValue = '"' . $value . '"';
+            } elseif (is_bool($value)) {
+                $formattedValue = $value ? 'true' : 'false';
+            } elseif (is_null($value)) {
+                $formattedValue = 'null';
+            } elseif (is_array($value)) {
+                // Check if it's an associative array (object) or indexed array
+                $isObject = array_keys($value) !== range(0, count($value) - 1);
+                if ($isObject) {
+                    $formattedValue = $this->getKotlinMapExampleRecursive($value, $indentLevel + 1);
+                } else {
+                    $formattedValue = $this->getArrayExample(json_encode($value), 'kotlin');
+                }
+            } else {
+                $formattedValue = (string)$value;
+            }
+            $mapEntries[] = $baseIndent . $formattedKey . ' to ' . $formattedValue;
+        }
+
+        if (count($mapEntries) > 0) {
+            $closeIndent = str_repeat('    ', $indentLevel + 1);
+            return "mapOf(\n" . implode(",\n", $mapEntries) . "\n" . $closeIndent . ")";
+        } else {
+            return 'mapOf( "a" to "b" )';
+        }
+    }
+
+    /**
+     * Generate Java-style map initialization using Map.of()
+     *
+     * @param array $data
+     * @return string
+     */
+    protected function getJavaMapExample(array $data): string
+    {
+        return $this->getJavaMapExampleRecursive($data, 0);
+    }
+
+    /**
+     * Recursive helper for generating Java Map.of() with proper indentation
+     *
+     * @param array $data
+     * @param int $indentLevel Indentation level for nested maps
+     * @return string
+     */
+    private function getJavaMapExampleRecursive(array $data, int $indentLevel): string
```

Comment:
Let's add default 0 for indentLevel, then we don't need the `getJavaMapExample` function